sale_margin_percentage/i18n/es.po

```diff
@@ -0,0 +1,78 @@
+# Translation of Odoo Server.
+# This file contains the translation of the following modules:
+#	* sale_margin_percentage
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: Odoo Server 11.0+e\n"
```

Comment:
Translations also needs migration. Take into account syntax changed for model terms from 11.0 -> 12.0.

Reply:
Change done.